<ui version="4.0" >
 <class>ChangeDataSampleDialog</class>
 <widget class="QDialog" name="ChangeDataSampleDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>295</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Change Data Samples</string>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>11</number>
   </property>
   <property name="topMargin" >
    <number>11</number>
   </property>
   <property name="rightMargin" >
    <number>11</number>
   </property>
   <property name="bottomMargin" >
    <number>11</number>
   </property>
   <property name="horizontalSpacing" >
    <number>6</number>
   </property>
   <property name="verticalSpacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" colspan="2" >
    <widget class="QLabel" name="TextLabel1" >
     <property name="text" >
      <string>Vectors to change:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <layout class="QVBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="_clear" >
       <property name="whatsThis" >
        <string>Unselect all vectors in the list.</string>
       </property>
       <property name="text" >
        <string>C&amp;lear</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="_selectAll" >
       <property name="whatsThis" >
        <string>Select all vectors in the list.</string>
       </property>
       <property name="text" >
        <string>S&amp;elect All</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <widget class="QListWidget" name="_curveList" >
     <property name="whatsThis" >
      <string>The sample ranges for the vectors selected in this list will be changed.</string>
     </property>
     <property name="selectionMode" >
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2" >
    <widget class="Kst::DataRange" native="1" name="_dataRange" />
   </item>
   <item row="3" column="0" colspan="2" >
    <widget class="QDialogButtonBox" name="_buttonBox" >
      <property name="orientation" >
        <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons" >
        <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
      </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Kst::DataRange</class>
   <extends>QWidget</extends>
   <header>datarange.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>_curveList</tabstop>
  <tabstop>_dataRange</tabstop>
  <tabstop>OK</tabstop>
  <tabstop>Cancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>Cancel</sender>
   <signal>clicked()</signal>
   <receiver>ChangeDataSampleDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OK</sender>
   <signal>clicked()</signal>
   <receiver>ChangeDataSampleDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
